### ISSUE TITLE
git: complete: Too many arguments

### ISSUE THREAD

**Issue by jakwings** [fish-shell/fish-shell#2603]
When I enter a wrong sub-command for git like `git x`, fish complains:

https://gist.github.com/anonymous/51237b482192389f82a3
```
complete: Too many arguments
~/.local/opt/fish/share/fish/completions/git.fish (line 299): complete -f -c git -n '__fish_git_using_command clone' -l bare 'Make a bare Git repository'
                                                              ^
from sourcing file ~/.local/opt/fish/share/fish/completions/git.fish
	called on standard input

in command substitution
	called on standard input


       complete ‐‐ edit command specific tab‐completions

   Synopsis
       complete ( ‐c | ‐‐command | ‐p | ‐‐path ) COMMAND
               [( ‐e | ‐‐erase )]
               [( ‐s | ‐‐short‐option ) SHORT_OPTION]
               [( ‐l | ‐‐long‐option | ‐o | ‐‐old‐option ) LONG_OPTION]
               [( ‐f | ‐‐no‐files )]
               [( ‐r | ‐‐require‐parameter )]
               [( ‐x | ‐‐exclusive )]
               [( ‐u | ‐‐unauthoritative )]
               [( ‐A | ‐‐authoritative )]
               [( ‐a | ‐‐arguments ) OPTION_ARGUMENTS]
               [( ‐w | ‐‐wraps ) WRAPPED_COMMAND]
               [( ‐n | ‐‐condition ) CONDITION]
               [( ‐d | ‐‐description ) DESCRIPTION]
       complete ( ‐C STRING | ‐‐do‐complete=STRING )

complete: Type 'help complete' for related documentation

…
```

**Comment by faho** [comment]
Already fixed by a0eef2663a6871673d3d9a9535fde704bd92c952. It seems you built right in the one day that was an issue. Sorry, my fault!

**Comment by jakwings** [comment]
Ok, see that. The hint is a little confusing without telling which arguments are invalid.
